### PR TITLE
docs: Update Welcome page and add to VCS

### PR DESCRIPTION
**Pull request by furi0us333**
- Implementing changes to the Welcome page and bringing that page into VCS
- Add supporting animations to the `assets` folder

### CODE REVIEW

**Review by cd-work — CHANGES_REQUESTED** [review]

File: docs/welcome/welcome.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+#### Phylum provides a comprehensive, scalable approach to defending your software supply chain. Get started with one or all of the below capabilities.
+
+# Detect & Prevent
+
+_Analyze risks and block threats from entering source code._
+
+- [Create a new Phylum project](https://docs.phylum.io/docs/phylum_project_create) in your project directory (`phylum project create <project-name>`)
+- [Submit your package lock file](https://docs.phylum.io/docs/phylum_analyze) (`phylum analyze <package-lock-file.ext>`)
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:25]: Will publish as is today, but plan to update with `phylum init` shortly as I get familiar with it.

File: docs/welcome/welcome.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- (Optional) View the analysis results in the [Phylum UI](https://app.phylum.io/auth/login)
+- https://blog.phylum.io/insights-and-resources/pick-a-python-lockfile-and-improve-security
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:27]: Relocated closer to the analyze command as that is where it is important to understand lockfiles (the reason they marketing wanted this post added)

File: docs/welcome/welcome.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Analyze Your First Project
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:29]: Agreed, removed the first section in favor of the complete walkthrough

File: docs/welcome/welcome.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### Download and Install the Phylum CLI
+
+The Phylum CLI is available for a [variety of operating systems on GitHub](https://github.com/phylum-dev/cli/releases/latest). However, you can easily install the CLI with the following:
+```sh
+curl <https://sh.phylum.io> | sh
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:35]: Removed

File: docs/welcome/welcome.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+phylum auth login
+```
+![Login](https://raw.githubusercontent.com/phylum-dev/cli/main/assets/cli-login.webp)
+
+### Create a New Phylum Project
+All analysis jobs *_must_* be associated with a Phylum project. To create a project, run the following:
+```sh
+phylum project create <projectName>
+```
+![Create Project](https://raw.githubusercontent.com/phylum-dev/cli/main/assets/create-project.webp)
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:50]: Will update to `phylum init` soon

File: docs/welcome/welcome.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+### Analyze a Project
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:52]: Updated

File: docs/welcome/welcome.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+To begin analyzing your project for software supply chain risks, submit a lockfile to Phylum.
+```sh
+phylum analyze <lockfile>
+```
+![Analyze Project](https://raw.githubusercontent.com/phylum-dev/cli/main/assets/submit-job.webp)
+
+### View Results
+After submitting your project, view the results in the [web UI](https://app.phylum.io).
+![View Results](https://raw.githubusercontent.com/phylum-dev/cli/main/assets/ui-project.webp)
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:61]: Updated it to read as an optional step

File: docs/welcome/welcome.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+# Extensions
+_Advance your OSS security by deploying your own extensions in the Phylum CLI (1-2 mins)_
+
+- [Extension Quickstart](https://docs.phylum.io/docs/extension_quickstart)
+- Generate your own extension skeleton
+  - `phylum extension new my-extension`
+  - Extensions always contain at least two files, the manifest describing the
+extension (PhylumExt.toml), and the entrypoint where the extension's execution will begin. Any additional source files can be included in the extension directory and imported from the entrypoint.
+  - The manifest file contains metadata about the extension beyond its executable source code. This allows you to define allowed resources in the sandboxed runtime. All available options can be found in the manifest format.
```
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/welcome/welcome.md:81]: Decided to remove this section as I don't believe Extensions belong on the Welcome page.

**Comment by furi0us333** [comment]
Closing this as the work was moved to another repo https://github.com/phylum-dev/documentation/pull/2

**Pull request closed without merging**